config/application.yml.example

```diff
@@ -58,11 +58,6 @@ development:
   system_satisfaction_survey: false
   editable_statuses: { 14: ['first_draft', 'draft', 'submitted', nil, 'get_a_cost_estimate', 'awaiting_pi_approval'],
                      48: ['draft', 'submitted'] }
```

Comment:
I moved the navigation links configurable from constant.yml to application.yml; modified the navbar partial and the index view in catalog_manager/catalog. I changed the name from navbar_links to nav_links as they are not limited to the navbar usage.